src/routes/join/+page.server.ts

```diff
@@ -35,15 +40,50 @@ export const actions = {
 		if (sessionData.tempIdExpiry.toDate() < new Date()) {
 			return fail(400, { tempId, expired: true });
 		}
-
 		// Add participant to session
 		await session.ref.update({
 			[`participants.${locals.user.uid}`]: {
 				name: locals.user.name,
-				joinedAt: new Date()
+				joinedAt: new Date(),
+				groupNumber: parseInt(groupNumber)
 			}
 		});
 
+		// Add participant to group
+		const groupSnapshot = await adminDb
+			.collection('groups')
+			.where('number', '==', parseInt(groupNumber))
+			.where('sessionId', '==', session.id)
+			.limit(1)
+			.get();
+
+		let groupRef;
```

Comment:
```suggestion
		let groupRef: FirebaseFirestore.DocumentReference | null = null;
```

Suggestions: 
- Define variable type
- Initial value